vcloud/box/common/bootstrap.erb

```diff
@@ -1,26 +1,31 @@
 #!/bin/bash
 
 if [ $1 = "postcustomization" ]; then
-  echo 'ubuntu:<%= vars[:pass_hash] %>' | chpasswd -e 
-  
+  echo 'ubuntu:<%= vars[:pass_hash] %>' | chpasswd -e
+
   dpkg-reconfigure openssh-server
   set -e
   exec >/var/log/userdata.log 2>&1
   export DEBIAN_FRONTEND=noninteractive
   E="/etc/environment"
   echo "LC_ALL=en_GB.UTF-8" > $E
-  
-  R="/etc/resolv.conf"
-  echo "nameserver 8.8.8.8" >$R
-  echo "nameserver 8.8.4.4" >>$R
-  
+
+  <% if vars[:dns_suffix] %>
+    sed -i '/^127.0.1.1/d' /etc/hosts
+    sed -i '/^dns-search/d' /etc/network/interfaces
+    sed -i '/^dns-nameservers/d' /etc/network/interfaces
+    echo -e 'dns-search\t<%= vars[:dns_suffix] %>\ndns-nameservers\t8.8.8.8 8.8.4.4' >> /etc/network/interfaces
+    echo -e 'nameserver 8.8.8.8\nnameserver 8.8.4.4\nsearch <%= vars[:dns_suffix] %>' | resolvconf -a eth0.inet
+  <% end %>
+
```

Comment:
Much of this is not required.

We added this block as a workaround for the fact that VMware Tools' guest customisation previously did not support Ubuntu Trusty LTS:
https://github.gds/gds/govuk-provisioning/commit/08679a6015d1b0246fba701db7d5a9a52ec802a4

We then later removed it because VMware Tools started supporting Trusty:
https://github.gds/gds/govuk-provisioning/commit/489a501c3865f010bdfd6039ad97715d639dc04d

Then added it back as it was necessary to override the domain in the FDQNs for the Draft stack, which no longer exists:
https://github.gds/gds/govuk-provisioning/commit/58538b52a972b79cbd1f9cb0b5fd38074b2ee950

I'll raise a PR to tidy up the govuk-provisioning repository, but in the meantime, most (if not all) of these lines can be left as-is.